pyat/at/collective/beam_loading.py

```diff
@@ -154,6 +154,8 @@ def __init__(self, family_name: str, length: float, voltage: float,
             cavitymode (CavityMode):  Is cavity ACTIVE (default) or PASSIVE
             buffersize (int):  Size of the history buffer for vbeam, vgen, vbunch
                 (default 0)
+            detune_angle:      Fixed detuning from optimal tuning angle. [rad]
```

Comment:
Should you describe the sign convention here?

Reply:
You may want to add this in the docstring of add_beamloading function as well no?